app/src/main/java/fr/free/nrw/commons/media/MediaDetailPagerFragment.java

```diff
@@ -415,4 +428,6 @@ public int getCount() {
             return provider.getTotalMediaCount();
         }
     }
+
+
```

Comment:
Would you mind undoing the addition of these two empty lines? Or was it on purpose maybe?

Reply:
no it was a mistake